### ISSUE TITLE
Add ability to reset requests counter

### ISSUE THREAD

**Issue by dewa1t** [mcustiel/phiremock-codeception-extension#14]
I prorose to add ability to reset requests counter during tests. If a suite has multiple cases with calls same mock it isn't easy to calculate expecting requests number in each case.

**Comment by mcustiel** [comment]
Hi, first of all. Thank you very much for all the contributions done in the last days. I am in to add more functionalities, but I have one question: what would be a use case for this that can not be achieved by verifying the amount of requests done?

**Comment by dewa1t** [comment]
The use case could be:
```
public function createTest($I)
{
    // Do call that uses external API stub here
    // Check count here, but it is 1 now and it will be 2 when updateTest is called.
}

/**
 * @before createTest
 */
public function updateTest($I)
{
    // Database state is reseted for each case due Codeception config
    // Call createTest in the scope of this test to get object to update
    // Do update
}
```

**Comment by dewa1t** [comment]
Othe case also could be:
```
public function someTest($I)
{
    // Check count is 1
}

public function otherTest($I)
{
    // Check count is 2
}

// Many cases here

public function laterOtherTest($I)
{
    // OMG, I forgot how many calls was. I'm so tired to calculate it each time.
}

public function newCaseThatShouldBeHereTest($I)
{
    // Damn! I've added this case and I should increase expected counters in all cases after!
}

// More cases here

**Comment by mcustiel** [comment]
Can not that be done by calling `$I->dontExpectRequestsInRemoteService();` in the before or after method? That resets phiremock requests. 
The use case I see is if the stubs are loaded from files, so it is a good functionality the one you propose. I like it.

**Comment by dewa1t** [comment]
I've tried, but `$I->dontExpectRequestsInRemoteService();` clears pre-defined expectations. (e.g. defined in json files), and I don't see ability to restore it with the module.

**Comment by mcustiel** [comment]
Released in version 1.4.0
Thank you @dewa1t
